manifests/charts/istio-operator/templates/clusterrole.yaml

```diff
@@ -91,6 +91,14 @@ rules:
   - rolebindings
   verbs:
   - '*'
+- apiGroups:
```

Comment:
Why is this needed? is the operator broken currently?

Reply:
yes, it failed because it did not have permission to get the lease